### ISSUE TITLE
go get github.com/denisenkom/go-mssqldb error

### ISSUE THREAD

**Issue by geosoft1** [denisenkom/go-mssqldb#229]
`go get github.com/denisenkom/go-mssqldb` give the following error

```
../../../../src/github.com/denisenkom/go-mssqldb/bulkcopy.go:171: cannot use tokenRow (type token) as type byte in argument to buf.WriteByte
../../../../src/github.com/denisenkom/go-mssqldb/bulkcopy.go:205: cannot use tokenDone (type token) as type byte in argument to buf.WriteByte
../../../../src/github.com/denisenkom/go-mssqldb/bulkcopy.go:240: cannot use tokenColMetadata (type token) as type byte in argument to buf.WriteByte

```

the package is still downloaded. i used this workaround until the issue is resolved:

- i removed from `github.com/denisenkom/go-mssqldb` `bulkcopy.go`,`bulkcopy_sql.go`,`bulkcopy_test.go`
- in mssql.go i commented the following lines (209-211)

```
	if len(query) > 10 && strings.EqualFold(query[:10], "INSERTBULK") {
		return c.prepareCopyIn(query)
	}

```

**Comment by wingyplus** [comment]
I already send pr https://github.com/denisenkom/go-mssqldb/pull/228 to fix this issue.

**Comment by AlekSi** [comment]
Related to #194.

**Comment by Fank** [comment]
same here

**Comment by adomokos** [comment]
Found it as well. `git reset --hard 9e40d9d5d325edfaa84d3374bfde6e1adce02d58` backed out the latest commits.

**Comment by wingyplus** [comment]
@adomokos I fix it in #228. Please try it.

Thanks

**Comment by kardianos** [comment]
Aged out.
